File: client/__tests__/__mocks__/tags.js
import faker from 'faker';

const tags = [] , tagsArraySize = 3;
for (let i = 0, l = tagsArraySize; i < l; i+=1) {
    tags.push({ 
        name: faker.internet.userName(),
        image: faker.image.imageUrl()
    });
}
export const tagsMock = tags;

File: spec/factories/tags.rb
# == Schema Information
#
# Table name: tags
#
#  id         :integer          not null, primary key
#  name       :string
#  created_at :datetime         not null
#  updated_at :datetime         not null
#  tag_image  :string
#  tag_icon   :string
#  tag_color  :string
#  thumbnail  :string
#

FactoryGirl.define do
  factory :tag do
    name { Faker::Lorem.word + SecureRandom.hex }
    tag_image { File.open("#{Rails.root}/spec/support/image.png", 'r') }

  end
end
